flatfilers/sandbox/src/index.ts

```diff
@@ -1,8 +1,25 @@
-import type { FlatfileListener } from '@flatfile/listener'
+import type { FlatfileListener, FlatfileEvent } from '@flatfile/listener'
 import { ExcelExtractor } from '@flatfile/plugin-xlsx-extractor'
 import { DelimiterExtractor } from '@flatfile/plugin-delimiter-extractor'
+import { automap } from "@flatfile/plugin-automap";
+
 
 export default async function (listener: FlatfileListener) {
   listener.use(ExcelExtractor())
   listener.use(DelimiterExtractor('txt', { delimiter: ',' }))
+
+  listener.use(
+    automap({
+      accuracy: "confident",
+      defaultTargetSheet: "contacts",
+      matchFilename: /test/,
+      debug: true,
+      onFailure: (event: FlatfileEvent) => {
+        // send an SMS, an email, post to an endpoint, etc.
```

Comment:
@damonbanks : Perhaps not entirely relevant, so please consider this as a question rather than a review.

Do we need to in any function to send the notification on failure?

await sendFailureNotification(event) ?
        
 Do we need a retry of any kind here?

Reply:
Good question. This method simply is in the flatfile sandbox folder, meaning that it isnt functional code. I left this in here for testing purposes.